### PR TITLE
F/marketo update

### PR DESCRIPTION
**Pull request by chrisjhoughton**
* bumped version 
* added company, sales person, opportunity, opportunity roles endpoints
* tested and running on staging

### CODE REVIEW

File: connectors/marketo/create_update_companies/schema.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+				text: 'Update only',
+				value: 'updateOnly'
+			}, {
+				text: 'Create or update',
+				value: 'createOrUpdate'
+			}, {
+				text: 'Create duplicate',
+				value: 'createDuplicate'
+			}],
+			default: 'createOrUpdate',
```
Comment (Johnbastian) [inline connectors/marketo/create_update_companies/schema.js]: Description?

File: connectors/marketo/create_update_leads/schema.js [hunk 2]
```diff
@@ -6,16 +6,31 @@
...
+				text: 'Update only',
+				value: 'updateOnly'
+			}, {
+				text: 'Create or update',
+				value: 'createOrUpdate'
+			}, {
+				text: 'Create duplicate',
+				value: 'createDuplicate'
+			}],
+			required: true,
```
Comment (Johnbastian) [inline connectors/marketo/create_update_leads/schema.js]: Description?

File: connectors/marketo/create_update_opportunities/schema.js [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+				value: 'updateOnly'
+			}, {
+				text: 'Create or update',
+				value: 'createOrUpdate'
+			}, {
+				text: 'Create duplicate',
+				value: 'createDuplicate'
+			}],
+			default: 'createOrUpdate',
+			required: true,
```
Comment (Johnbastian) [inline connectors/marketo/create_update_opportunities/schema.js]: Description?

File: connectors/marketo/create_update_opportunity_roles/schema.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+				value: 'updateOnly'
+			}, {
+				text: 'Create or update',
+				value: 'createOrUpdate'
+			}, {
+				text: 'Create duplicate',
+				value: 'createDuplicate'
+			}],
+			default: 'createOrUpdate',
+			required: true,
```
Comment (Johnbastian) [inline connectors/marketo/create_update_opportunity_roles/schema.js]: Description?

File: connectors/marketo/create_update_sales_persons/schema.js [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+				value: 'updateOnly'
+			}, {
+				text: 'Create or update',
+				value: 'createOrUpdate'
+			}, {
+				text: 'Create duplicate',
+				value: 'createDuplicate'
+			}],
+			default: 'createOrUpdate',
+			required: true,
```
Comment (Johnbastian) [inline connectors/marketo/create_update_sales_persons/schema.js]: Description?

File: connectors/marketo/list_object_fields_ddl/model.js [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+	expects: function (res) {
+		if (!res.body.result || !res.body.result.length) {
+			return when.reject(new Error('No company descriptions were returned.'));
+		}
+	},
+
+	afterSuccess: function (body) {
+		return {
+			result: _.map(body.result[0].fields, function (field) {
```
Comment (Johnbastian) [inline connectors/marketo/list_object_fields_ddl/model.js]: What if there are more than 1 results?
Might want to copy and tweak the `afterSuccess` from _list_object_filter_types_ddl/model.js_

Comment (chrisjhoughton) [inline connectors/marketo/list_object_fields_ddl/model.js]: There's nothing to suggest in the API docs that a "described" object will every have more than one result here. It's super weird, but it makes no sense even having this as an array in the first place - so it's acceptable to take the first item in my view.

Comment (Johnbastian) [inline connectors/marketo/list_object_fields_ddl/model.js]: I just checked the API. Why are you using `.fields` instead of `.searchableFields` attribute?

Comment (Johnbastian) [inline connectors/marketo/list_object_fields_ddl/model.js]: Ok, ignore this.

**Pull request merged**